bin/snare

```diff
@@ -202,8 +201,6 @@ if __name__ == '__main__':
         loop.run_forever()
     except (KeyboardInterrupt, TypeError) as e:
         loop.run_until_complete(app.stop())
+        compare_version_proc.terminate()
         print(e)
```

Comment:
Could you please remove this print? :) I spend some time trying to understand why I still have error in console :D